lang/test/org/partiql/lang/eval/visitors/StaticTypeInferenceVisitorTransformTest.kt

```diff
@@ -7402,6 +7407,99 @@ class StaticTypeInferenceVisitorTransformTest : VisitorTransformTestBase() {
 
             return incompatibleTypeForIndex + validIndexType
         }
+        private fun createAggFunctionValidTests(
+            functionName: String,
+            inputTypes: StaticType,
+            expectedType: StaticType,
+            isTopLevel: Boolean
```

Comment:
Just curious does being a top level value matter for aggregation functions? When I looked at the tests, the results look the same.